plover/gui/add_translation.py

```diff
@@ -24,6 +24,10 @@ def __init__(self, parent, engine, config):
                            pos, wx.DefaultSize, 
                            wx.DEFAULT_DIALOG_STYLE, wx.DialogNameStr)
 
+        transparency = config.get_translation_frame_transparency()
+
+        self.SetTransparent(255 - (2.55 * transparency))
```

Comment:
SetTransparent takes an int but we'll be passing in a float this way. Can we make sure to explicitly pass an int?

A very minor note: just int'ing only yields [0, 254] on my system but rounding gives the full [0, 255]:

self.SetTransparent(255 - int(round(2.55 * transparency)))